src/services/venue-service.ts

```diff
@@ -48,6 +70,68 @@ export class VenuesService  {
       Id: 3,
       name: 'Super long startup name that doesnt fit on the screen in one line',
       address: '2120 Market St.',
+      latitude: 39.7545,
+      longitude: -104.99217,
+      url: 'https://skookum.com/',
+      mapIcon: require('../images/map-icons/supperLongName-map-icon.svg'),
+      companyImage: require('../images/company-images/skookum.png'),
+      description: `
+      Signpost is a smart CRM that helps businesses to know, grow, and strengthen their local
+      customer base. Mia automatically collects and analyzes consumer data from every touchpoint,
+      including email, calls and transactions.
+      `,
+    },
+    {
+      Id: 4,
+      name: 'Marketo',
+      address: '707 17th St.',
+      latitude: 39.7471025,
+      longitude: -104.9907957,
+      url: 'https://skookum.com/',
+      mapIcon: require('../images/map-icons/supperLongName-map-icon.svg'),
+      companyImage: require('../images/company-images/skookum.png'),
+      description: `
+      Signpost is a smart CRM that helps businesses to know, grow, and strengthen their local
+      customer base. Mia automatically collects and analyzes consumer data from every touchpoint,
+      including email, calls and transactions.
+      `,
+    },
+    {
+      Id: 5,
+      name: 'Gusto',
+      address: '1201 16th St',
+      latitude: 39.7491778,
+      longitude: -104.9968469,
+      url: 'https://skookum.com/',
+      mapIcon: require('../images/map-icons/supperLongName-map-icon.svg'),
+      companyImage: require('../images/company-images/skookum.png'),
+      description: `
+      Signpost is a smart CRM that helps businesses to know, grow, and strengthen their local
+      customer base. Mia automatically collects and analyzes consumer data from every touchpoint,
+      including email, calls and transactions.
+      `,
+    },
+    {
+      Id: 6,
+      name: 'WeWork',
+      address: '2420 17th St, Denver, CO 80202',
+      latitude: 39.7586051,
+      longitude: -105.0074086,
+      url: 'https://skookum.com/',
+      mapIcon: require('../images/map-icons/supperLongName-map-icon.svg'),
+      companyImage: require('../images/company-images/skookum.png'),
+      description: `
+      Signpost is a smart CRM that helps businesses to know, grow, and strengthen their local
+      customer base. Mia automatically collects and analyzes consumer data from every touchpoint,
+      including email, calls and transactions.
+      `,
+    },
+    {
+      Id: 7,
+      name: 'Wurk',
```

Comment:
Wurk 🤣

Reply:
Haha right, its a real company in Denver though!  I know the guy that runs it, and they're kicking butt.  HR software.  We'll probably see them participating in DSW.
https://www.enjoywurk.com/